stl/src/vector_algorithms.cpp

```diff
@@ -15,7 +15,7 @@
 #include <cstdint>
 #include <emmintrin.h>
 #include <immintrin.h>
-#include <intrin0.h>
+#include <intrin.h>

```

Comment:
`<intrin.h>` is guaranteed to include `<immintrin.h>` and `<emmintrin.h>`, so we don't need these fine-grained headers anymore.